appeals/api/src/server/endpoints/documents/documents.service.js

```diff
@@ -55,7 +56,7 @@ export const getFolderForAppeal = async (appeal, folderId) => {
  */
 export const getFoldersForAppeal = async (appeal, stage = null) => {
 	if (stage && stage != null) {
-		const paths = getFoldersForStage(stage);
+		const paths = getFoldersForStage(appeal.appealType, stage);
```

Comment:
No need to pass the appeal type

Reply:
I added this to support the API returning different folders depending on the appeal type, but it's been removed now.